ray-operator/config/rbac/role.yaml

```diff
@@ -6,6 +6,15 @@ metadata:
   creationTimestamp: null
   name: kuberay-operator
 rules:
+- apiGroups:
```

Comment:
Could you explain the changes to this file? Were these autogenerated?

Reply:
Auto generated and some manual fix to keep all existing rules.

Reply:
em.. chenk008 used to update the coordination for leadership election in https://github.com/ray-project/kuberay/pull/169/files. Seems the current script override the older fashion.  What's the kubebuilder version are you using?

Reply:
Default plugin keys: "go.kubebuilder.io/v3"
Default project version: "3"
controller-gen.kubebuilder.io/version: v0.6.0

I think I am using the same version